wordpress-mobile/WordPress-Android
# Login Rework: wire up the help action buttons on login flows

**Note: #6398 needs to be merged first.**

This PR implements the "HELP" actionbar button by opening the already available help options screen.

To test (1):
* On any of the login flow screens, tap the "HELP action button. The blue help options screen should appear.

To test (2):
* On the site address login screen, tap the "Need help finding your site address" secondary action. A popup should appear.
* Tap the "NEED MORE HELP?" button on the popup. The blue help options screen should appear.
